modulefiles/hafs.hera.lua

```diff
@@ -121,7 +121,7 @@ cdo_ver=os.getenv("cdo_ver") or "1.9.8"
 load(pathJoin("cdo", cdo_ver))
 
 rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"
-load(pathJoin("rocoto", rocoto_ver))
+load("rocoto")
```

Comment:
@BijuThomas-NOAA, with the change to use `load("rocoto")`, do we still need to keep the line above `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`? Or alternatively, will `rocoto_ver=os.getenv("rocoto_ver") or "default"` work? Thanks!

Reply:
@BinLiu-NOAA:
We do not need to keep the line `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`. 
I do not think `rocoto_ver=os.getenv("rocoto_ver") or "default"` will work

Reply:
RT passed on Orion

Reply:
> @BijuThomas-NOAA, with the change to use `load("rocoto")`, do we still need to keep the line above `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`? Or alternatively, will `rocoto_ver=os.getenv("rocoto_ver") or "default"` work? Thanks!

We do not keep the line `rocoto_ver=os.getenv("rocoto_ver") or "1.3.3"`
I do not think `rocoto_ver=os.getenv("rocoto_ver") or "default"` will work